app/timetables/models.py

```diff
@@ -175,3 +175,16 @@ def __str__(self):
 
     class Meta:
         unique_together = ('timetable', 'cycle_day', 'meal', 'meal_option')
+
+
+class Vendor(SlugifyMixin, TimestampMixin):
+    """Model representing food service-provider"""
+
+    name = models.CharField(max_length=255)
+    slug = models.SlugField(max_length=255, unique=True, editable=False)
```

Comment:
It is not a URL slug field though could be used for that purpose later on. Name field is desired to be unique retaining the format it is entered by the user. To achieve that, uniqueness is enforced on a slugified version instead. For instance, `quaker oats` is a duplicate of `Quaker Oats` with unified slug of `quaker-oats`.